data/autoyast_sle12/autoinst_sles12sp5_media_minimal_x86.xml

```diff
@@ -0,0 +1,1661 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <add-on>
+    <add_on_products config:type="list"/>
+  </add-on>
+  <bootloader>
```

Comment:
We can compact this part refer to https://openqa.nue.suse.com/tests/11164010/settings/autoyast_sle15/mini.xml

Ex:
<bootloader>
        <global>
            <timeout config:type="integer">-1</timeout>
        </global>
    </bootloader>